### DISCUSSION TITLE
Setup suggestion for Intraday Trading Stocks

### DISCUSSION THREAD

**Discussion by sunel** [pranjal-joshi/Screeni-py#108] [General]
@pranjal-joshi any suggestion for Intraday setup

This artical has good explanation https://www.kotaksecurities.com/ksweb/intraday-trading/how-to-choose-stocks-for-intraday-trading

**Comment by pranjal-joshi** [comment]
Currently not trading in cash segment for Intraday. You can experiment with Index  F&O.
No plans for integrating F&O trades in screeni-py so far.
